docs/src/modules/components/AppFrame.js

```diff
@@ -66,9 +71,7 @@ function DeferredAppSearch() {
         as="style"
       />
       {/* Suspense isn't supported for SSR yet */}
-      {typeof window === 'undefined' ? (
```

Comment:
I had a feeling that this was terribly hack. Something similar broke next in 16.10.0. It's now properly supported in 16.10.1 but the [semantics for it are not intended](https://github.com/facebook/react/issues/16938#issuecomment-536157631) (and https://github.com/facebook/react/issues/16938#issuecomment-536207871).

Better get rid of it now and render the suspense boundary unconditionally once suspense is supported on the server.